backend/tests/test_project.py

```diff
@@ -99,15 +192,15 @@ def test_get_project_by_id(self):
         )
         p = Project(
             id=p_id,
+            type="dummy type",
+            contacts=[c],
```

Comment:
@peterghrong Is it possible for you to revert it back as these are unnecessary changes.